### DISCUSSION TITLE
[ 功能建议 ] 实例列表只列出用户所在资源组实例，并隐藏binlog清理按钮

### DISCUSSION THREAD

**Discussion by peng19832** [hhyo/Archery#1610] [General]
### 目前遇到的问题/使用障碍
只要赋予用户菜单实例列表权限，就会列出所有的实例，且会展示binlog清理按钮，存在安全隐患。

### 希望如何解决/实现它
只列出该用户所在资源组内的实例，同时隐藏binlog清理按钮。

### 其他信息
![image](https://user-images.githubusercontent.com/9590209/141260191-c7e50c6d-1c81-41ef-aa40-630330509179.png)

**Comment by hhyo** [comment]
这个最初就没有考虑非管理员的查看，没有相应的权限控制

**Comment by peng19832** [comment]
> 这个最初就没有考虑非管理员的查看，没有相应的权限控制

如果非管理员，想了解下自己可用的实例有哪些，就只有找管理员了，有点绕。
有些时候他们还是需要看一下的
